code/modules/spells/ability_types/realized.dm

```diff
@@ -436,3 +436,78 @@
 		L.physiology.white_mod /= 2
 		L.physiology.black_mod /= 2
 		L.physiology.pale_mod /= 2
+
+/mob/living/simple_animal/hostile/farmwatch_plant//TODO: give it an effect with the corresponding suit.
+	name = "Tree of Desires"
+	desc = "The growing results of your research."
+	health = 60
+	maxHealth = 60
+	icon = 'ModularTegustation/Teguicons/tegumobs.dmi'
+	icon_state = "farmwatch_tree"
+	icon_living = "farmwatch_tree"
+	icon_dead = "farmwatch_tree"
+	del_on_death = FALSE
+
+/mob/living/simple_animal/hostile/farmwatch_plant/Move()
+	return FALSE
+
+/mob/living/simple_animal/hostile/farmwatch_plant/CanAttack(atom/the_target)
+	return FALSE
+
+/mob/living/simple_animal/hostile/farmwatch_plant/Initialize()
+	. = ..()
+	QDEL_IN(src, 15 SECONDS)
+
+/mob/living/simple_animal/hostile/farmwatch_plant/death()
+	density = FALSE
+	animate(src, alpha = 0, time = 1)
+	QDEL_IN(src, 1)
+	..()
+
+/mob/living/simple_animal/hostile/spicebush_plant
```

Comment:
I believe if these don't have their faction set to `list("neutral")` they'll be ignored by most enemies.

Reply:
done, for both of the summons.